ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift

```diff
@@ -26,7 +26,8 @@ open class GatewayAPI: NSObject, NSCoding {
         return self.gatewayAddress.absoluteString
     }
 
-    private var accessToken: String?
+    /** Access token of this gate way */
+    open private(set) var accessToken: String?
```

Comment:
@rizakii 

Your point is "Why does not use public" isn't it?
If so, because All other properties are open. I arrange to other properties.

open properties can be override by extended class. In discussion with kumano-san, We adopted to forgive to override properties and methods in principle.

What do you think?